internal/pkg/service/buffer/api/service/service.go

```diff
@@ -34,18 +34,14 @@ import (
 type service struct {
 	deps   dependencies.ForServer
 	mapper mapper.Mapper
-	stats  statistics.Manager
+	stats  *statistics.APINode
 }
 
 func New(d dependencies.ForServer) buffer.Service {
 	return &service{
 		deps:   d,
 		mapper: mapper.NewMapper(d.BufferAPIHost()),
-		stats: statistics.New(d, func(ctx context.Context, stats []model.SliceStats) {
-			if err := d.Store().UpdateSliceStats(ctx, stats); err != nil {
-				d.Logger().Error("cannot update slice stats in etcd: %s", err.Error())
-			}
-		}),
+		stats:  statistics.NewAPINode(d),
```

Comment:
Uz v teste vieme pouzit `MocedDependencies`, tak to vieme ostestovat aj priamo so store + etcd.